pkg/blob/controllerserver.go

```diff
@@ -292,6 +316,10 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		}
 	}
 
+	if createPrivateEndpoint {
+		setKeyValueInMap(parameters, serverNameField, fmt.Sprintf("%s.blob.%s", accountName, storageEndpointSuffix))
```

Comment:
pls also write a PrivateEndpoint e2e test for blobfuse and nfs if both are supported.

Reply:
this may be wrong, for azure file private end point support, there is `privatelink` in `serverNameField`:
```
	if createPrivateEndpoint {
		setKeyValueInMap(parameters, serverNameField, fmt.Sprintf("%s.privatelink.file.%s", accountName, storageEndpointSuffix))
	}
```

Reply:
According to the [documentation](https://learn.microsoft.com/en-us/azure/storage/common/storage-private-endpoints?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json#dns-changes-for-private-endpoints), if private endpoint is created for the blob, then `accountname.blob.core.windows.net` will be resolved to `accoutname.privatelink.blob.core.windows.net` as CNAME, so we are still using private endpoint in this case. If not, the storage firewall will deny the public traffic since we already configure the network rules for the storage account.

Reply:
in that case, then we don't need to set `accountname.blob.core.windows.net` for serverName?